doorpi/action/SingleActions/mailto.py

```diff
@@ -39,6 +43,18 @@ def fire_action_mail(smtp_to, smtp_subject, smtp_text):
         msg['Subject'] = doorpi.DoorPi().parse_string(smtp_subject)
         msg.attach(MIMEText(doorpi.DoorPi().parse_string(smtp_text), 'html'))
         msg.attach(MIMEText('\nsent by:\n'+doorpi.DoorPi().epilog, 'plain'))
+
+ 	#add a snapshot
+        if (smtp_snapshot and len(doorpi.DoorPi().config.get('SIP-Phone', 'capture_device', '')) > 0):
+            file = createSnapshot()
+	    if (len(file) > 0):
```

Comment:
den verstehe ich nicht - die Frage ist doch nicht die Länge, sondern is not False
Oder willst Du eine List mit mehreren Snapshots übergeben und mehrere Bilder der Mail anhängen?